OpenOversight/app/main/views.py

```diff
@@ -214,8 +215,14 @@ def get_officer():
     )
 
 
-@sitemap_include
 @main.route("/label", methods=[HTTPMethod.GET, HTTPMethod.POST])
+def labels_redirect():
+    flash(FLASH_MSG_PERMANENT_REDIRECT)
+    redirect(url_for("get_started_labeling"), code=HTTPStatus.PERMANENT_REDIRECT)
```

Comment:
| Description | Permanent  | Temporary |
| ------------- | ------------- | ------------- |
| Allows changing the request method from POST to GET   | 301  | 302  |
| Doesn't allow changing the request method from POST to GET  | 308  | 307  |

Link: https://stackoverflow.com/a/42138726